Logic/Path/dijkstra_graph.py

```diff
@@ -42,7 +48,7 @@ def find_any_goal_path(self, start, goals):
         return []
 
     def find_a_goal_path(self, initial, end):
-        '''Responsible for calculating the optimal path for a goal'''
+        '''Responsible for calculating the optimal path for a goal using dijkstra algorithm'''
```

Comment:
Should Dijkstra be mentioned here as well? It might seem repetitive (I know I mentioned this earlier but I am realising it might not be necessary).

Reply:
Done